Anchor.example.toml

```diff
@@ -9,7 +9,7 @@ cluster = "localnet"
 wallet = <Your Key Path Here>
 
 [scripts]
-test = "yarn run ts-mocha -p ./tsconfig.json -t 1000000 tests/**/*.ts"
+test = "yarn run ts-mocha -p --parallel ./tsconfig.json -t 1000000 tests/**/*.ts"
 
 [test.validator]
 url = "mainnet"
```

Comment:
I think we have to update these to the current accounts below: `8nzxsNf74ZHDguHi51SjQWxDxegL2DBgxeGHA2pQVtTJ` & `K2z1qkxZdsw6WpFd63hqhqx9MYUc5c85NdbXULNeGhW`